nixos/modules/security/auditd.nix

```diff
@@ -170,6 +170,38 @@ in
         n: v: lib.nameValuePair "audit/audisp-${n}.conf" { text = prepareConfigText v.config; }
       ) (lib.filterAttrs (n: v: v.config != null) cfg.plugins));
 
+    security.auditd.plugins = {
+      af_unix = {
+        path = lib.getExe' pkgs.audit "audisp-af_unix";
+        args = [
+          "0640"
+          "/var/run/audispd_events"
+          "string"
+        ];
+        format = "binary";
+      };
+      remote = {
+        path = lib.getExe' pkgs.audit "audisp-remote";
+        config = { };
+      };
+      filter = {
+        path = lib.getExe' pkgs.audit "audisp-filter";
+        args = [
+          "allowlist"
+          "/etc/audit/audisp-filter.conf"
+          (lib.getExe' pkgs.audit "audisp-syslog")
+          "LOG_USER"
+          "LOG_INFO"
+          "interpret"
+        ];
+        config = { };
+      };
+      syslog = {
+        path = lib.getExe' pkgs.audit "audisp-syslog";
+        args = [ "LOG_INFO" ];
+      };
+    };
```

Comment:
This configuration is not introspectable from the options search. Reconfiguring the module from the config block is not ideal.

Reply:
Yes, but i am not super sure how to do this better. I still want to allow for arbitrary plugins to be configurable, and these default plugins need to be configured *somewhere*.

Reply:
I have no good answer for how to do this any other way, sorry.

Reply:
I think this is a good enough solution for now and definitely a significant improvement over the status quo.